pkg/apis/ceph.rook.io/v1/keys.go

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2018 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+const (
+	KeyMon       = "mon"
```

Comment:
I think it could be good to create a type for the keys and have all the keys be of that type. The linter/compiler would help us make sure the keys aren't misused.
```go
type KeyType string

const (
    KeyMon KeyType = "mon"
    ...
)
```